src/Stratis.Bitcoin.IntegrationTests/ProofOfStakeSteps.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using NBitcoin;
+using Stratis.Bitcoin.Features.Consensus;
+using Stratis.Bitcoin.Features.Miner.Interfaces;
+using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.Features.Wallet.Controllers;
+using Stratis.Bitcoin.Features.Wallet.Models;
+using Stratis.Bitcoin.IntegrationTests.Builders;
+using Stratis.Bitcoin.IntegrationTests.EnvironmentMockUpHelpers;
+using static Stratis.Bitcoin.Features.Miner.PosMinting;
+
+namespace Stratis.Bitcoin.IntegrationTests
+{
+    public class ProofOfStakeSteps
+    {
+        private IDictionary<string, CoreNode> nodes;
+        private NodeGroupBuilder nodeGroupBuilder;
+        private SharedSteps sharedSteps;
+
+        private HdAddress powSenderAddress;
+        private Key powSenderPrivateKey;
+
+        private HdAddress posReceiverAddress;
+        private Key posReceiverPrivateKey;
+
+        public readonly string PowMiner = "ProofOfWorkNode";
```

Comment:
should all these be constants? and why public ?  addressed from outside?

Reply:
Constants now and yes used outside the class.

Reply:
Addressed outside and need to be read only only fields as calling code returns "Member 'ProofOfStakeSteps.PosStaker' cannot be accessed with an instance reference; qualify it with a type name instead", when constants.